### DISCUSSION TITLE
Project Needler

### DISCUSSION THREAD

**Discussion by Reiers** [FilOzone/Space-Panthers#9] [Ideas]
## Needler 

The primary objective of this project is to establish a comprehensive testing ground that will be readily accessible for FilDev via SSH at any given time.

The testing ground will serve multiple purposes, including but not limited to:

**- Testing Lotus**: Ensuring the functionality and reliability of Lotus.
**- Testing Boost**: Evaluating the performance and capabilities of Boost.
**- Feature Testing**: Probing new and existing features for stability and efficiency.
**- Hardware Testing**: Assessing the compatibility and performance of various hardware configurations.
**- TSE Maintenance**: Regular maintenance and updates by the TSE team to ensure the testing ground remains state-of-the-art.
**- Automatic Testing:** Implementing automated test scripts and routines to facilitate continuous testing.
**- Create Grafana Panels**: Designing and integrating Grafana panels for real-time monitoring and analytics.
**- Side Projects**: Allowing room for experimental projects and other side endeavors that can benefit from a robust testing environment.

Current layout for testing (Draft):

![ground_control_diagram](https://github.com/FILCAT/Space-Panthers/assets/63351350/8aa1f387-3b74-40a1-a9c9-7029658a61ce)

**Comment by jennijuju** [comment]
could you please add the size of nodes you are targeting for each network?

**Reply by Reiers** [reply]
### Storage servers and workers, can be added as needed. 

![Power](https://github.com/FILCAT/Space-Panthers/assets/63351350/444493c5-0999-442e-878f-63d9749be898)
